### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by phil-davis**

### CODE REVIEW

File: operator/runner/runner.go [hunk 1]
```diff
@@ -295,7 +295,7 @@ func (r *Runner) Run(ctx context.Context, id int64) error {
 		),
 		transform.WithEnviron(environ),
 		transform.WithEnviron(r.Environ),
-		transform.WithLables(
+		transform.WithLabels(
```
Comment (phil-davis) [inline operator/runner/runner.go:298]: This one is strange - I didn't find any other mention of the typo string `WithLables` so I am not sure how `transform.WithLables()` ever worked???

File: service/hook/parser/testdata/gitea_push.json [hunk 2]
```diff
@@ -17,25 +17,25 @@
     "Sha": "4522cbcefc20728a5b72b3a86af35e608622c514",
     "Message": "Updated readme\n",
     "Author": {
-      "Name": "Unknwon",
```
Comment (phil-davis) [inline service/hook/parser/testdata/gitea_push.json:20]: Were all these typos of "Unknwon" intentional?

File: operator/runner/runner.go (hunk 1 above)
Comment (phil-davis) [reply operator/runner/runner.go]: Hmmm - the build failed, so I had to revert this change/ that typo is needed.

File: service/hook/parser/testdata/gitea_push.json (hunk 2 above)
Comment (tboerger) [reply service/hook/parser/testdata/gitea_push.json:20]: That's not a typo, Unknwon is the name of the author if Gogs ;)

Comment (phil-davis) [reply service/hook/parser/testdata/gitea_push.json]: I  removed all those "unknwon" changes.

File: operator/runner/runner.go (hunk 1 above)
Comment (tboerger) [reply operator/runner/runner.go]: https://github.com/drone/drone-yaml/blob/master/yaml/compiler/transform/labels.go#L21 there you can see the wrong function name ;)

But AFAIK this code is anyway dead.

**Comment by phil-davis** [comment]
I adjusted/reduced the lines fixed, to not include the various "intended typos" mentioned in the review comments. This is ready to review again.

**Comment by tphoney** [comment]
Hey @phil-davis thanks for the many many typo fixes. Much appreciated. 

Cheers,
TP

**Pull request merged**